modules/engage-paella-player/package-lock.json

```diff
@@ -2961,9 +2961,9 @@
       "dev": true
     },
     "lunr": {
-      "version": "2.3.6",
-      "resolved": "https://registry.npmjs.org/lunr/-/lunr-2.3.6.tgz",
-      "integrity": "sha512-swStvEyDqQ85MGpABCMBclZcLI/pBIlu8FFDtmX197+oEgKloJ67QnB+Tidh0340HmLMs39c4GrkPY3cmkXp6Q==",
+      "version": "2.3.7",
```

Comment:
It's says 2.3.7 here but this lunr is not used in the target paella compile. Not sure it matters. If the reviewer thinks it's confusing, it can be changed to match the one in the paella shrinkwrap.